packages/vats/src/vat-bank.js

```diff
@@ -480,7 +480,15 @@ const prepareBank = (
         const balanceUpdater = makeBalanceUpdater(brand, publisher);
 
         // Abort if we lost the race to another call.
-        addressToUpdater.init(address, balanceUpdater);
+        if (addressToUpdater.has(address)) {
+          // XXX should this be an exception?
```

Comment:
@dckc @michaelfig any danger in allowing this?

Reply:
I don't know off-hand, and I'm struggling to figure it out.

But if we do allow it, make sure to update the comment. The call to `makeBalanceUpdater` should probably go inside the `if` clause too.

Reply:
Yes, it is working as designed.  The existing code throws if the address key is already present in the collection.  If you're going to try to make the exception more explanatory, then please don't change the semantics.

But I like the terseness of 1 comment and 1 line of code instead of more than 8 to accomplish the same thing.

Reply:
Note these new lines are behaving differently: if `addressToUpdate.has(address)` it's logging a warning instead of throwing an exception. The code comment was to ask if that's okay. 

It sounds like this looser behavior is not okay. I'm interested in learning more about why.

Meanwhile, I suppose I'll have need to find a different way to get the test to not call this `.init` repeatedly.